File: Modules/Common/Items/SIE.Web.Items/ProductModels/Commands/ProductModelSkillImportCommand.cs
using SIE.Common.ImportHelper;
using SIE.Items.ProductModels;
using SIE.Web.Command;
using SIE.Web.Common.Import.Commands;
using System;
using System.Data;

namespace SIE.Web.Items.ProductModels.Commands
{
    /// <summary>
    /// 机型技能导入
    /// </summary>
    [JsCommand("SIE.Web.Items.ProductModels.Commands.ProductModelSkillImportCommand")]
    public class ProductModelSkillImportCommand : ImportCommandBase
    {
        /// <summary>
        /// 获取导入完成后逻辑
        /// </summary>
        /// <returns></returns>
        protected override ImportCompleted GetImportCompleted()
        {
            return (DataRow[] drSuccess, DataRow[] drFailed) =>
            {

            };
        }

        /// <summary>
        /// 获取导入处理类型
        /// </summary>
        /// <returns></returns>
        protected override Type GetImportHandleType()
        {
            return typeof(ProductModelSkillImportHandle);
        }
    }
}

File: Modules/Common/Items/SIE.Web.Items/ProductModels/Commands/ProductModelSkillImportCommand.js
SIE.defineCommand('SIE.Web.Items.ProductModels.Commands.ProductModelSkillImportCommand', {
    extend: 'SIE.Web.Common.Import.Commands.ImportCommandBase',
    meta: { text: "导入", group: "business", iconCls: "icon-Upload icon-blue" },
    canExecute: function (view) {
        if (view.getParent() == null || view.getParent().getCurrent() == null) return false;
        return true;
    }
});